### DISCUSSION TITLE
typescript format or file `web-dev-server.config.ts`?

### DISCUSSION THREAD

**Discussion by aelbore** [modernweb-dev/web#1616] [Q&A]
can i create a typescript file of `web-dev-server.config.ts`? as of it is only `.js` and `.mjs` file extension

**Comment by LarsDenBakker** [comment]
You can create it and and compile it with TSC. There is no built-in compiler for ts files in WDS.

**Comment by ahnpnl** [comment]
May I ask if there is a plan to support it? If we want to have it support, how should we contribute?
